tools/assimp_view/MessageProc.cpp

```diff
@@ -107,7 +107,7 @@ void MakeFileAssociations() {
         RegCreateKeyEx(HKEY_CURRENT_USER,buf,0,NULL,0,KEY_ALL_ACCESS, NULL, &hRegistry,NULL);
         RegSetValueEx(hRegistry,"",0,REG_SZ,(const BYTE*)"ASSIMPVIEW_CLASS",(DWORD)strlen("ASSIMPVIEW_CLASS")+1);
         RegCloseKey(hRegistry);
-    } while ((sz = strtok(NULL,";")));
+    } while ((sz = strtok(NULL,";")) != nullptr);
```

Comment:
Change the other `NULL` too.

Reply:
@turol 

Do you mean, to change only the three **NULL** in the `RegCreateKeyEx` call above the highlighted line, or in the entire project?

Changing every **NULL** reference in the project would add 326 +- changes to the diff. I would recommend to make such changes in a separate PR because it goes beyond the scope of the current PR, which is to simply fix the compile warnings.